tests/helm/testdata/goldenfile/services_with_service_monitor_labels/services_with_service_monitor_labels/default.output.yaml

```diff
@@ -1,136 +1,4 @@
 ---
-# Source: sumologic/templates/events/common/service.yaml
```

Comment:
Why are these resources removed from this file? Were they unused, and regenerating the file caused them to disappear?

Reply:
I think something is wrong and only one object is put into the generated file